src/components/gitOps/constants.ts

```diff
@@ -95,3 +96,16 @@ export const gitOpsRepoNotConfiguredWithEnforcedEnv = (env: string): string =>
     `Deployment to ‘${env}’ requires a GitOps repository. Please configure and try again.`
 export const gitOpsRepoNotConfigured =
     'GitOps repository is required to deploy using GitOps. You can deploy using helm or configure GitOps repository and try again.'
+
+export const getDocLinkForGitProvider = (currentGitProvider: GitProvider): string => {
+    if (currentGitProvider === GitProvider.GITLAB) {
```

Comment:
we can make an `object` for this

this would be scable and will be more readable